cross-media-measurement/src/test/kotlin/org/wfanet/measurement/common/RenewedFlowTest.kt

```diff
@@ -32,7 +32,7 @@ class RenewedFlowTest {
   @Test
   fun `flow repeats`() = runBlockingTest {
     var i = 1
-    val result = renewedFlow<Int>(1000, 0) {
+    val result = renewedFlow(1000, 0) {
```

Comment:
Can you annotate the type of result here to make it a little more clear in case the reader isn't totally familiar with renewedFlow?